tree/ntuple/v7/doc/specifications.md

```diff
@@ -768,24 +768,32 @@ e.g. `std::vector<MyEvent>` or `std::vector<std::vector<float>>`.
 
 ### Fundamental Types
 
-The following fundamental types are stored as `leaf` fields with a single column each:
-
-| C++ Type                         | Default RNTuple Column | Alternative Encoding  |
------------------------------------|------------------------|-----------------------|
-| bool                             | Bit                    |                       |
-| char                             | Char                   |                       |
-| int8_t                           | Int8                   |                       |
-| uint_8_t, unsigned char          | UInt8                  |                       |
-| int16_t                          | SplitInt16             | Int16                 |
-| uint16_t                         | SplitUInt16            | UInt16                |
-| uint32_t                         | SplitUInt32            | UInt32                |
-| int32_t                          | SplitInt32             | Int32                 |
-| uint64_t                         | SplitUInt64            | UInt64                |
-| int64_t                          | SplitInt64             | Int64                 |
-| float                            | SplitReal32            | Real32                |
-| double                           | SplitReal64            | Real64                |
+The following fundamental types are stored as `leaf` fields with a single column each.
+Type can potentially be stored in multiple possible column types.
+The possible combinations are marked as `W` in the following table
+Additionally, some types allow for reading from certain column types but not to write into them.
+Such cases are marked as `R` in the table.
+
+|               |                                                  Fundamental C++ Type                                                   ||
```

Comment:
double `|` at EOL?

Reply:
Supposedly there is a markdown dialect where the right double pipe is used to merge cells. Github doesn't seem to understand that though *sigh*